tests/test_LSF_Status_Checker.py

```diff
@@ -0,0 +1,273 @@
+import unittest
+from unittest.mock import patch
+from tests.src.lsf_status import LSF_Status_Checker, BjobsError
+from tests.src.OSLayer import OSLayer
+from subprocess import CalledProcessError
+
+count_fail_three_times_and_then_return_DONE=0
+count_fail_three_times_and_then_return_PEND=0
+count_fail_three_times_and_then_return_FAIL=0
```

Comment:
Do these need to be global? They're only used once each?

Reply:
They are used only once and don't need to be global. I was having issues with this, and I just left the first thing that worked. Refactored at https://github.com/leoisl/snakemake-lsf/commit/8a259bb089e1085f05673cf4054ff3b1fd26d170